src/ui/controls/edit.cpp

```diff
@@ -2353,8 +2359,11 @@ void CEdit::MoveChar(int move, bool bWord, bool bSelect)
         }
         else
         {
-            m_cursor1 ++;
-            if ( m_cursor1 > m_len )  m_cursor1 = m_len;
+            if ( m_cursor1 < m_len )
+            {
+                m_cursor1 ++;
+                while ( m_cursor1 < m_len && (m_text[m_cursor1] & 0xC0) == 0x80 )  m_cursor1 ++;
```

Comment:
We use C++17 now, so I'd also replace the hex values with binary ones so it's easier to see what's going on.

Reply:
How to name this function?

Reply:
This is not entirely fair, because the code already contains hex values.

Reply:
I am not asking for changing the whole file so how is it unfair?

Reply:
Master branch uses C++11. I did not notices that dev uses C++17. Binary constants definitely look more clear in that case.